src/main/java/sdk/sample/Cleanup.java

```diff
@@ -0,0 +1,184 @@
+package sdk.sample;
+
+import com.azure.resourcemanager.netapp.fluent.NetAppManagementClient;
+import com.azure.resourcemanager.netapp.fluent.models.CapacityPoolInner;
+import com.azure.resourcemanager.netapp.fluent.models.NetAppAccountInner;
+import com.azure.resourcemanager.netapp.fluent.models.VolumeInner;
+import sdk.sample.common.CommonSdk;
+import sdk.sample.common.ProjectConfiguration;
+import sdk.sample.common.Utils;
+import sdk.sample.model.ModelCapacityPool;
+import sdk.sample.model.ModelNetAppAccount;
+import sdk.sample.model.ModelVolume;
+
+public class Cleanup {
+    /**
+     * Breaks and removes Data Replication connection and then deletes all resources -> volumes, pools and accounts
+     * @param config Project Configuration
+     * @param anfClient Azure NetApp Files Management Client
+     */
+    public static void runCleanup(ProjectConfiguration config, NetAppManagementClient anfClient)
+    {
+        if (config == null || config.getAccounts() == null)
+            return;
+
+        /*
+          Break and remove data replications
+         */
+        Utils.writeConsoleMessage("Breaking and removing Data Replication(s)...");
+        for (ModelNetAppAccount account : config.getAccounts())
```

Comment:
would be great to have config.getAccounts() as a global variable

Reply:
Done